net-misc/asterisk/asterisk-18.24.1.ebuild

```diff
@@ -330,13 +328,23 @@ src_install() {
 }
 
 pkg_postinst() {
-	if [ -z "${REPLACING_VERSIONS}" ]; then
+	if [[ -z "${REPLACING_VERSIONS}" ]]; then
 		elog "Asterisk Wiki: https://wiki.asterisk.org/wiki/"
 		elog "Gentoo VoIP IRC Channel: #gentoo-voip @ irc.libera.chat"
-	elif [ "$(ver_cut 1 "${REPLACING_VERSIONS}")" != "$(ver_cut 1)" ]; then
-		elog "You are updating from Asterisk $(ver_cut 1 "${REPLACING_VERSIONS}") upgrade document:"
-		elog "https://wiki.asterisk.org/wiki/display/AST/Upgrading+to+Asterisk+$(ver_cut 1)"
-		elog "Gentoo VoIP IRC Channel: #gentoo-voip @ irc.libera.chat"
+	else
+		local my_replacing=() x
+		for x in ${REPLACING_VERSIONS}; do
+			[[ "$(ver_cut 1 "${x}")" != "$(ver_cut 1)" ]] && \
```

Comment:
Still unnecessary quoting and backslash.

Reply:
I've removed all quoting here now since none of it is technically required (${x} cannot contain spaces here).

Your original variation still had quotes around ${x} in some places, and around $() in others - how do I decide when it should be present and when not?